### PR TITLE
Add additional config option to append a CI skip to commit messages

### LINKED ISSUE #105: Boomper should support adding [skip ci] to its commit message
We don't want to invoke a CI build every time Boomper bumps a version in our README. Including `[skip ci]` in the commit message would let these commits skip the unnecessary build. 

It'd be nice to have this configurable in case there was a need to do CI build (although I can't think of a good use case).

### PR DESCRIPTION
**Pull request by mikeandersen**
If you specify multiple files, Boomper creates a commit for each file which can trigger a lot of redundant CI builds.

This allows specifying a config option to append a common CI skip annotation to the commit message. Defaults to `false` so it should be backwards compatible with existing installations.

Resolves #105

### CODE REVIEW

**Comment by jessebye** [comment]
@toolmantim any chance of getting this merged? It'd save us a lot of wasted build time 😁

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ You can configure Boomper using the following key in your `.github/boomper.yml`
 |`updates.[].pattern`|Required|The regular expression containing a single group, which will be used to match and update the version number in the file.|
 |`updates.[].branch`|Optional|The branch to update. Default is the repository's default branch (e.g. `master`).|
 |`branches`|Optional|The branches to listen for configuration updates to `.github/boomper.yml`. Useful if you want to test the app on a pull request branch. Default is `"master"`.|
+|`skipCi`|Optional|Appends `[ci skip]` to commit messages to prevent triggering additional CI builds. Default is `false`.|
```
Comment (toolmantim) [inline README.md:62]: Could we make this `skip-ci` please?

**Comment by mikeandersen** [comment]
@toolmantim Thanks, made the requested change. 🙂

**Comment by toolmantim** [comment]
Thanks @mikeandersen!

**Pull request merged**